app/views/profile/partials/_form_profile.blade.php

```diff
@@ -1,7 +1,7 @@
   {{-- Birthdate --}}
   <div class="field-group -dual -alpha {{ setInvalidClass('birthdate', $errors) }}">
     {{ Form::label('birthdate', 'Birthdate: ') }}
-    {{ Form::input('date', 'birthdate', null, ['placeholder' => 'MM/DD/YYYY']) }}
+    {{ Form::text('birthdate', null, ['placeholder' => 'MM/DD/YYYY']) }}
```

Comment:
I think we want this to remain a `date` field type for validation reasons, I _think_ this was just to remove the silly html5 picker thing if we can